### PR TITLE
add divider

### PR DESCRIPTION
**Pull request by Weii-Dev9**
## Overview
- Reference card [#12](https://trello.com/c/BzAAVrad/12-1-每条朋友圈之间加上分割线)

Why was this PR created / what was done?

- add divider between each content

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality not to work as expected)

## Checklist
1. [ ] Makes your PR pass tests?
2. [x] PR attached to trello card
3. [x] Trello card moved into 'Review'

### CODE REVIEW

File: app/src/main/java/com/thoughtworks/androidtrain/helloworld/compose/TweetsScreen.kt [hunk 1]
```diff
@@ -72,35 +76,31 @@ fun TweetsScreen() {
 
 @Composable
 private fun TweetItem(tweet: Tweet) {
-    val flag = remember { mutableStateOf(false) }
-    Row(modifier = Modifier.padding(all = 8.dp)) {
+    Row(modifier = Modifier.padding(16.dp)) {
```
Comment (yunlongwen) [inline app/src/main/java/com/thoughtworks/androidtrain/helloworld/compose/TweetsScreen.kt]: 和颜色一样，应该把值定义在 res下。

File: app/src/main/java/com/thoughtworks/androidtrain/helloworld/compose/TweetsScreen.kt [hunk 2]
```diff
@@ -72,35 +76,31 @@ fun TweetsScreen() {
...
         val painter = rememberAsyncImagePainter(tweet.sender?.avatar)
-        //val painter = painterResource(R.mipmap.avatar)
         Avatar(painter)
 
-        Spacer(modifier = Modifier.width(8.dp))
+        Spacer(modifier = Modifier.width(16.dp))
 
-        Column {
+        Column(horizontalAlignment = Alignment.Start) {
+            val nickColor = Color(0xFF4e7fe4)
```
Comment (yunlongwen) [inline app/src/main/java/com/thoughtworks/androidtrain/helloworld/compose/TweetsScreen.kt]: 建议用应用的调色板定义颜色值，不要再代码中写value。
我看你的应用中已经定义好了一些颜色，请仔细check这些颜色的，应该保持代码风格统一。

File: app/src/main/java/com/thoughtworks/androidtrain/helloworld/compose/TweetsScreen.kt [hunk 3]
```diff
@@ -59,8 +56,15 @@ fun TweetsScreen() {
...
-                TweetItem(tweet)
+            items(tweets) { tweet ->
+                if (tweet.content != "" && tweet != tweets.last()) {
+                    TweetItem(tweet)
+                    Divider(
+                        color = Color.LightGray,
+                        thickness = 0.5.dp,
+                        modifier = Modifier.fillMaxWidth()
+                    )
+                }
```
Comment (yunlongwen) [inline app/src/main/java/com/thoughtworks/androidtrain/helloworld/compose/TweetsScreen.kt:67]: 代码注释中写的  `//顶部  //内容 //底部`

我的理解是你想把这个页面分成三部分，那么中间内容部分应该考虑封装成 `@Composable`函数。

**Comment by yunlongwen** [comment]
Hey @Weii-Dev9 

请关注PR 模板信息，review代码前应该全部填写(或者✅)。

WeChatWorkScreenshot_1e90a275-a246-4a89-a623-c8c74538e18a

File: app/src/main/java/com/thoughtworks/androidtrain/helloworld/compose/TweetsScreen.kt (hunk 3 above)
Comment (Weii-Dev9) [reply app/src/main/java/com/thoughtworks/androidtrain/helloworld/compose/TweetsScreen.kt:67]: LazyColumn{}组件自身分了三部分，内容部分的items()是LazyColumn的方法，传一个list进行渲染

**Review by yunlongwen — APPROVED** [review]

**Pull request merged**